platform/ios/PolyPodApp/PolyPod/OnboardingView.swift

```diff
@@ -131,31 +131,8 @@ private struct Slide: View {
             Spacer()
             
             if let confirmLabel = confirmLabel {
-                Button(action: confirmAction ?? {}) {
-                    Text(confirmLabel)
-                        .font(.custom("Jost-Medium", size: 14))
-                        .kerning(-0.18)
-                        .foregroundColor(Color.PolyPod.darkForeground)
-                        .frame(minWidth: 296, minHeight: 48)
-                        .background(
-                            RoundedRectangle(cornerRadius: 4)
-                                .fill(Color(fromHex: "#FB8A89"))
-                                .shadow(
-                                    color: Color.black.opacity(0.06),
-                                    radius: 2,
-                                    x: 0,
-                                    y: 1
-                                )
-                                .shadow(
-                                    color: Color.black.opacity(0.1),
-                                    radius: 3,
-                                    x: 0,
-                                    y: 1
-                                )
-                        )
-                }
-                .buttonStyle(PlainButtonStyle())
-                .frame(maxWidth: .infinity, alignment: .center)
+                PrimaryButton(title: confirmLabel,
```

Comment:
No change of logic here, just moving the Button somewhere else, right?

Reply:
Right, moved to be reused.